app/analysis/templates/analysis.html

```diff
@@ -1,119 +1,91 @@
-<!DOCTYPE html>
+{% extends "base.html" %}
+
 {% load static %}
 
-<html>
-  <head>
-    <link rel="stylesheet" type="text/css" href="{% static 'bootstrap/css/bootstrap.min.css' %}" />
-    <link rel="stylesheet" type="text/css" href="{% static 'global_style.css' %}" />
-    <link rel="stylesheet" type="text/css" href="{% static 'analysis.css' %}" />
-    <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
-    <script src="https://code.highcharts.com/highcharts.js"></script>
-    <script src="https://code.highcharts.com/highcharts-3d.js"></script>
-    <script src="https://code.highcharts.com/modules/exporting.js"></script>
-    <script src="https://d3js.org/d3.v4.min.js"></script>
-    <script scr="{% static 'd3-axis.js' %}"></script>
-    <script src="{% static 'd3tips.js' %}"></script>
-    <script src="{% static 'd3matrix.js' %}"></script>
-    <script src="{% static 'bootstrap/js/bootstrap.min.js' %}"></script>
-    <script src="{% static 'global_js.js' %}"></script>
-    <script src="{% static 'analysis.js' %}"></script>
+{% block page %} | Analysis{% endblock %}
 
-    <title>EDAM</title>
-  </head>
+{% block custom_refs %}
+<link rel="stylesheet" type="text/css" href="{% static 'analysis.css' %}" />
+<script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/highcharts-3d.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://d3js.org/d3.v4.min.js"></script>
```

Comment:
Can you change it so we grab bootstrap and d3 from twitter and cloudfare or someone else? It'll save our users a lil bandwidth.